source/utilities.tex

```diff
@@ -5414,7 +5414,7 @@
 \pnum
 Implementations should avoid the use of dynamically allocated memory for a small contained object.
 \begin{example}
-where the object constructed is holding only an \tcode{int}.
+Where the object constructed is holding only an \tcode{int}.
```

Comment:
Could you please leave this untouched in your change and open an edit issue for rephrasing this?